CONTRIBUTING.md

```diff
@@ -127,6 +127,12 @@ Once we've discussed your changes and you've got your code ready, make sure that
     * When the test file is too large, for example > 800 lines, you should split it to a new file.
       Please take a look at `t/plugin/limit-conn.t` and `t/plugin/limit-conn2.t`.
 
+## Contributor T-shirt
+
+If you have contributed to Apache APISIX, no matter it's code contributions to fix a bug and add functionality, or documentation and evangelistic contributions to make Apache APISIX be understood and used by more developers, congratulations! You are eligible to receive the very special Contributor T-shirt! Go ahead and fill out the [Contributors Submissions form](https://docs.google.com/forms/d/e/1FAIpQLSdXEpCs60UK49UlOGdBCQSXr7DYz3enyT4GJPKrYQmYfVLPKQ/viewform).
```

Comment:
```suggestion
If you have contributed to Apache APISIX, no matter it is a code contribution to fix a bug or a feature request, or a documentation change, Congratulations! You are eligible to receive the very special Contributor T-shirt! It's always been the community effort that has made Apache APISIX be understood and used by more developers. Go ahead and fill out the [Contributors Submissions form](https://docs.google.com/forms/d/e/1FAIpQLSdXEpCs60UK49UlOGdBCQSXr7DYz3enyT4GJPKrYQmYfVLPKQ/viewform).
```

Why do you use `evangelistic`  : ), it's "seeking to convert others to the Christian faith; missionary."